firestore-bigquery-export/firestore-bigquery-change-tracker/src/bigquery/checkUpdates.ts

```diff
@@ -0,0 +1,74 @@
+import { Table, TableMetadata } from "@google-cloud/bigquery/build/src/table";
+import { Partitioning } from "./partitioning";
+
+import { FirestoreBigQueryEventHistoryTrackerConfig } from ".";
+
+export async function tableRequiresUpdate(
+  table: Table,
+  config: FirestoreBigQueryEventHistoryTrackerConfig,
+  schemaFields: any,
+  documentIdColExists: boolean,
+  pathParamsColExists: boolean
+): Promise<boolean> {
+  // If the table doesn't exist, return early.
+  if (!table) {
```

Comment:
The type here is `table: Table` which suggests this will never be called

Reply:
Agreed! Will remove.